griptape/drivers/sql/snowflake_sql_driver.py

```diff
@@ -65,6 +68,6 @@ def get_table_schema(self, table: str, schema: Optional[str] = None) -> Optional
             metadata_obj = sqlalchemy.MetaData()
             metadata_obj.reflect(bind=self.engine)
             table = sqlalchemy.Table(table, metadata_obj, schema=schema, autoload=True, autoload_with=self.engine)
-            return str([(c.name, c.type) for c in table.columns])
+            return str([(c.name, c.type) for c in getattr(table, "columns")])
```

Comment:
What's the error that prompted this change?

Reply:
Cannot access member "columns" for type "str"

Reply:
Can you try this? https://stackoverflow.com/questions/24959589/get-table-columns-from-sqlalchemy-table-model